packages/components/modules/messages/common/graphql/queries/GroupDetailsQuery.ts

```diff
@@ -1,13 +1,13 @@
 import { graphql } from 'react-relay'
 
 export const GroupDetailsQuery = graphql`
-  query GroupDetailsQuery($roomId: ID!) {
+  query GroupDetailsQuery($roomId: ID!, $q: String) {
     chatRoom(id: $roomId) {
       id
       participantsCount
       ...GroupTitleFragment
-      ...MembersListFragment
-      ...ProfileSummaryFragment
+      ...MembersListFragment @arguments(q: $q)
+      ...ProfileSummaryFragment @arguments(q: $q)
```

Comment:
it seems weird to have `MembersListFragment` and `ProfileSummaryFragment` listed here, and they all have essentially the same fields, I think we should fully remove `ProfileSummaryFragment` from the code base, and also look into removing `RoomTitleFragment`, we should move this logic to the backend and treat group and 1-1 room's name and avatar the same way on the frontend.

@anicioalexandre what do you think?

Reply:
@nossila  super agreed, I noticed we have some repeated fragments that we could merge too